### DISCUSSION TITLE
v0.10.0(重要重要重要)

### DISCUSSION THREAD

**Discussion by lanyeeee** [lanyeeee/bilibili-manga-downloader#59] [Announcements]
`portable`为免安装版，解压后双击直接运行，容易被杀毒软件(如Windows Defender)误判为病毒

[关于被杀毒软件误判为病毒](https://github.com/lanyeeee/bilibili-manga-downloader?tab=readme-ov-file#%E5%85%B3%E4%BA%8E%E8%A2%AB%E6%9D%80%E6%AF%92%E8%BD%AF%E4%BB%B6%E8%AF%AF%E5%88%A4%E4%B8%BA%E7%97%85%E6%AF%92)

### 新增

- 解决下载付费章节会封号的问题 #57

<hr /><em>This discussion was created from the release <a href='https://github.com/lanyeeee/bilibili-manga-downloader/releases/tag/v0.10.0'>v0.10.0</a>.</em>

**Comment by redwhale-q** [comment]
win10系统，exe和portable都试过。exe安装后报错【应用程序发生异常unknown software exception(0x80000003),位置为0x00007FFAEEB850D0.要终止程序，请单击“确定”。】portable双击或者以管理员状态运行，都会闪退跳不出界面。

**Reply by lanyeeee** [reply]
只有v0.10.0会这样吗？之前的版本能打开吗(不要使用，看到图形界面就行)

**Reply by redwhale-q** [reply]
发现我之前0.9的版本也是打开就闪退，但我没有报杀毒

**Reply by lanyeeee** [reply]
去水印的工具你应该也打不开吧？

**Reply by lanyeeee** [reply]
装这个后再试试
[MicrosoftEdgeWebview2Setup.zip](https://github.com/user-attachments/files/17902334/MicrosoftEdgeWebview2Setup.zip)

**Reply by redwhale-q** [reply]
感谢！我修复了一下webview，现在已经很流畅地在使用了，非常感谢您的开发！

**Comment by huamanfei** [comment]
不行，哥，还是被封了。

**Reply by huamanfei** [reply]
最新的版本，刚下了一本，五六十话吧。

**Reply by redwhale-q** [reply]
被封是账号永久封禁还是只被封一段时间？

**Comment by huamanfei** [comment]
30天，小事。只是给你反应一下，软件很好用，想你能完善一下。
还有一个小发现，这次这个版本一次只能下一个，所以下了三四十话就封了。上次可以多线程，下了100多话才警告的。
上次是警告，这次封了也正常。
你先完善软件，一个月后我再下。哈哈哈！





&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "lanyeeee/bilibili-manga-downloader"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年11月25日(星期一) 晚上7:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [lanyeeee/bilibili-manga-downloader] v0.10.0(重要重要重要) (Discussion #59)





 
被封是账号永久封禁还是只被封一段时间？

**Reply by lanyeeee** [reply]
你反映封号后我又下了一整本电锯人(第一季+第二季前86话)，还是没能触发封号。
也许是你的账号体质特殊？你有注意把cookie复制完整吗？

**Reply by huamanfei** [reply]
应该是完整，可能上次警告被盯上了。行，你先接着完善，我申请解封了，不过应该解不了，只能下个月再下载了。

**Reply by huamanfei** [reply]
还有，这次这个版本怎么没有多线程了。

**Reply by lanyeeee** [reply]
多线程下载太快容易封号，现在严打。
以前B漫是完全不管的，所以想下多快就下多快。

**Comment by huamanfei** [comment]
这个Cookie每次刷新都不一样吗，我发现每次刷新都不一样。我应该是复制完整了，我感觉主要原因是我前几天刚被警告了，被重点照顾了。
&nbsp;





&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年11月25日(星期一) 晚上7:34
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [lanyeeee/bilibili-manga-downloader] v0.10.0(重要重要重要) (Discussion #59)














 
多线程下载太快容易封号，现在严打。
 以前B漫是完全不管的，所以想下多快就下多快。

**Reply by lanyeeee** [reply]
会有略微不同，但不影响，只要某一次的完整cookie就行
应该是账号被重点关照了

**Comment by huamanfei** [comment]
安装失败，说已经安装了。反正我账号被封了一个月。你先看下大家的反应，如果没事我一个月后再下。&nbsp;





&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年11月25日(星期一) 晚上7:43
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [lanyeeee/bilibili-manga-downloader] v0.10.0(重要重要重要) (Discussion #59)














 
装这个后再试试
 MicrosoftEdgeWebview2Setup.zip

**Reply by lanyeeee** [reply]
你看错消息了，我那条消息是回别人的，他无法打开软件
你能正常打开，提示已经安装了是符合预期的，这个东西是前置，一般系统默认自带的
